WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterFragment.kt

```diff
@@ -31,6 +39,35 @@ class ActivityLogTypeFilterFragment : DialogFragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        return inflater.inflate(layout.activity_log_type_filter_fragment, container, false)
+        return inflater.inflate(R.layout.activity_log_type_filter_fragment, container, false)
+    }
+
+    override fun getTheme(): Int {
+        return R.style.WordPress_FullscreenDialog
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initToolbar(view)
+        initRecyclerView()
+    }
+
+    private fun initToolbar(view: View) {
+        val toolbar = view.findViewById(R.id.toolbar_main) as MaterialToolbar
+        toolbar.navigationIcon = ColorUtils.applyTintToDrawable(
+                toolbar.context, R.drawable.ic_close_white_24dp,
+                toolbar.context.getColorResIdFromAttribute(R.attr.colorOnSurface)
+        )
+        toolbar.setNavigationContentDescription(R.string.close_dialog_button_desc)
+        toolbar.setNavigationOnClickListener { dismiss() }
+    }
+
+    private fun initRecyclerView() {
+        recycler_view.layoutManager = LinearLayoutManager(context, RecyclerView.VERTICAL, false)
```

Comment:
Question: Is there a reason why setting a layout manager is not done on the xml level instead?

Reply:
I don't think there is any specific reason, but we mostly use this programatic approach throughout the codebase.